homeassistant/components/unifiprotect/config_flow.py

```diff
@@ -0,0 +1,218 @@
+"""Config Flow to configure UniFi Protect Integration."""
+from __future__ import annotations
+
+import logging
+from typing import Any
+
+from aiohttp import CookieJar
+from pyunifiprotect import NotAuthorized, NvrError, ProtectApiClient
+from pyunifiprotect.data.nvr import NVR
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import (
+    CONF_HOST,
+    CONF_ID,
+    CONF_PASSWORD,
+    CONF_PORT,
+    CONF_USERNAME,
+    CONF_VERIFY_SSL,
+)
+from homeassistant.core import callback
+from homeassistant.data_entry_flow import FlowResult
+from homeassistant.helpers.aiohttp_client import async_create_clientsession
+
+from .const import (
+    CONF_ALL_UPDATES,
+    CONF_DISABLE_RTSP,
+    CONF_OVERRIDE_CHOST,
+    DEFAULT_PORT,
+    DEFAULT_VERIFY_SSL,
+    DOMAIN,
+    MIN_REQUIRED_PROTECT_V,
+)
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class ProtectFlowHandler(config_entries.ConfigFlow, domain=DOMAIN):
+    """Handle a UniFi Protect config flow."""
+
+    VERSION = 2
```

Comment:
Why 2 instead of 1?

Reply:
This is a port of the HACS integration, so that means it should be compatible with it. The HACS integration is on version 2.